package.json

```diff
@@ -25,6 +26,7 @@
         "react-reveal": "^1.2.2",
         "react-scripts": "3.0.1",
         "rewire": "^4.0.1",
+        "shortid": "^2.2.14",
```

Comment:
Do not use a library for this. I have a hash generating function, you can also write it from scratch. Make sure you carefully consider when you add libraries to achieve a task. It can add unnecessary bloat and security risks.